mobile-client/app/.gitignore

```diff
@@ -1 +1,14 @@
 /build
+# User-specific configurations
+.idea
+.idea/libraries/
+.idea/workspace.xml
+.idea/tasks.xml
+.idea/.name
```

Comment:
instead of listing all these files like this, just put ```.idea/``` and it will ignore all subfiles.